server/db/routing/config.js

```diff
@@ -180,8 +180,10 @@ const leaveCrew = require('./../utils/user_crewHelpers').leaveCrew;
 router.delete('/user/crews', (req, res) => {
   let userId = req.body.id;
   let crewId = req.body.crewId;
+  console.log('userid', userId, "crewId", crewId)
```

Comment:
could you remove these console.logs? I did change the test so that it will fail if this breaks for some reason.